### PR TITLE
Feature/Update nav drawer

### PR DESCRIPTION
**Pull request by friedger**
The Drawer now contains the image of the Home GDG (or the GDG logo).

It is still ugly but works, sorry for being a developer.

device-2015-02-28-114244

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java [hunk 1]
```diff
@@ -90,10 +111,50 @@ public void onDrawerOpened(View drawerView) {
         };
         mDrawerLayout.setDrawerListener(mDrawerToggle);
 
+        String homeChapterId = getSharedPreferences("gdg", MODE_PRIVATE).getString(Const.SETTINGS_HOME_GDG, null);
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java]: Changing the old code is hard but I think we should stop using Strings like "gdg" for the future.

File: app/src/main/java/org/gdg/frisbee/android/adapter/DrawerAdapter.java [hunk 2]
```diff
@@ -40,8 +40,9 @@ private void initAdapter() {
             }
             add(new DrawerItem(Const.DRAWER_ACHIEVEMENTS, R.drawable.ic_drawer_achievements, R.string.achievements));
             add(new DrawerItem(Const.DRAWER_ARROW, R.drawable.ic_drawer_arrow, R.string.arrow));
-            add(new DrawerItem(Const.DRAWER_SETTINGS, R.drawable.ic_drawer_settings, R.string.settings));
-            add(new DrawerItem(Const.DRAWER_ABOUT, R.drawable.ic_drawer_about, R.string.about));
+            add(new DrawerItem(Const.DRAWER_SETTINGS, android.R.drawable.ic_menu_preferences, R.string.settings));
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/adapter/DrawerAdapter.java]: Are you sure about this? android.R version of drawables are changing very frequenlty and different in all platform versions

File: app/src/main/java/org/gdg/frisbee/android/fragment/InfoFragment.java [hunk 3]
```diff
@@ -290,7 +253,55 @@ public void onNotFound(String key) {
...
+                    mResourcesBox.addView(tv);
+                }
+
+            }
+            mFetchOrganizerInfo.buildAndExecute();
+        }
+    }
+
+    private Person getPerson(final String[] params) {
+        try {
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/fragment/InfoFragment.java]: DUplicate code detected :)

File: app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java (hunk 1 above)
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java]: Acknowledged

File: app/src/main/java/org/gdg/frisbee/android/adapter/DrawerAdapter.java (hunk 2 above)
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/adapter/DrawerAdapter.java]: That was the quickest solution I could come up with

File: app/src/main/java/org/gdg/frisbee/android/fragment/InfoFragment.java (hunk 3 above)
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/fragment/InfoFragment.java]: Yes, https://github.com/gdg-x/frisbee/issues/245

File: app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java (hunk 1 above)
Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/activity/GdgNavDrawerActivity.java]: https://github.com/gdg-x/frisbee/issues/246

**Comment by HenriqueRocha** [comment]
Build is failing with:
> * What went wrong:
> A problem was found with the configuration of task ':app:packageAlpha'.
> File '/home/travis/build/gdg-x/frisbee/settings' specified for property 'signingConfig.storeFile' is not a file.

Do you know how to fix this?

**Comment by friedger** [comment]
updated the screenshot

**Comment by friedger** [comment]
Not sure how to deal with too long list of items in the drawer. We would like to invite the users to scroll if necessary...

**Comment by HenriqueRocha** [comment]
I already started rewriting the drawer as a ScrollView instead of a Listview just like in the Google I/O app. I'll take care of that, don't worry.

**Comment by friedger** [comment]
It is a scrollable list view. How does the user know that it is scrollable?
It is the same for scrollview?
Am 28.02.2015 14:15 schrieb "Henrique Rocha" <notifications@github.com>:

**Comment by HenriqueRocha** [comment]
The user knows that it is scrollable because it's a known pattern on
Android in the navigation drawer. It's scrollable on Gmail, Play Store,
etc.
On 28 Feb 2015 14:18, "Friedger Müffke" <notifications@github.com> wrote:

> It is a scrollable list view. How does the user know that it is scrollable?
> It is the same for scrollview?
> Am 28.02.2015 14:15 schrieb "Henrique Rocha" <notifications@github.com>:
>
> > I already started rewriting the drawer as a ScrollView instead of a
> > Listview just like in the Google I/O app. I'll take care of that, don't
> > worry.
> >
> > --
> > Reply to this email directly or view it on GitHub
> > <https://github.com/gdg-x/frisbee/pull/244#issuecomment-76525645>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/244#issuecomment-76525758>.
>

-- 
Disclaimer: The information in this e-mail and any attachments is the 
property of Novoda Ltd and is confidential and may be legally privileged. 
This e-mail is intended solely for the person or organisation to which it 
is addressed. Any disclosure, copying or other use of the information by 
any person or organisation who is not the intended recipient is strictly 
prohibited and may be unlawful. If you have received this e-mail in error, 
please inform the sender immediately and delete/destroy this e-mail and any 
copies of it. Novoda Ltd has taken reasonable precautions to minimise the 
risk of any software viruses which may damage your systems, but we advise 
that you take the necessary steps to ensure that no virus contamination is 
suffered. Novoda Ltd does not accept any liability for any loss or damage 
caused by the transmission of any virus.

Novoda Ltd, Company No: 347444, Registered in Scotland Registered 
Office: C/O Alexander Sloan, 38 Cadogan Street, Glasgow, G2 7HF, Scotland.  VAT 
Registration Number  GB 984 2525 93

**Pull request merged**